src/Avalonia.Controls/Control.cs

```diff
@@ -126,6 +126,15 @@ public FlowDirection FlowDirection
             set => SetValue(FlowDirectionProperty, value);
         }
 
+        /// <summary>
+        /// Occurs when an element wishes to be scrolled into view.
+        /// </summary>
+        public event EventHandler<ContextRequestedEventArgs>? RequestBringIntoView
```

Comment:
Again, kind of an internal event though this one does have potential use-cases in user code so I'm OK with exposing this on as a CLR event.

Reply:
Reverted changes